packages/tables/src/styled/StyledCell.ts

```diff
@@ -21,18 +22,64 @@ export interface IStyledCellProps {
   hasOverflow?: boolean;
   /** The width of the cell */
   width?: string | number;
+  size: SIZE;
 }
 
-export const StyledCell = styled.td.attrs<IStyledCellProps>(props => ({
+const truncatedStyling = css`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+`;
+
+const sizeStyling = (props: IStyledCellProps & ThemeProps<DefaultTheme>) => {
+  let padding = `${props.theme.fontSizes.xs} ${props.theme.fontSizes.sm}`;
+  const verticalPadding = math(`(${getRowHeight(props)} - ${getLineHeight(props)}) / 2`);
+  let rightPadding;
+  let width = props.width;
+  let height;
+
+  if (props.hasOverflow) {
+    padding = '0';
+    width = '2em';
+    height = 'inherit';
+  }
+
+  if (props.isMinimum) {
+    if (props.theme.rtl) {
+      rightPadding = `${props.theme.space.base * 3}px`;
+    }
+
+    rightPadding = '0';
+    width = '1em';
+  }
+
+  return css`
+    padding: ${padding};
+    /* stylelint-disable declaration-block-no-redundant-longhand-properties */
+    padding-top: ${verticalPadding};
+    padding-right: ${rightPadding};
+    padding-bottom: ${verticalPadding};
```

Comment:
The design mocks indicate that the top and bottom spacing should each be `10px`. I notice that on the "Overflow menu" example, the bottom spacing on each row appears to be larger than the top, and more than `10px`:

<img width="955" alt="Screen Shot 2020-01-07 at 3 16 07 PM" src="https://user-images.githubusercontent.com/1811365/71937268-c16d2b00-3160-11ea-915a-1025e1163143.png">

Reply:
When refactoring the `verticalPadding` and `padding` values I didn't handle the `hasOverflow` prop correctly. This is now corrected.